common/src/main/java/org/apache/rocketmq/common/PlainAccessConfig.java

```diff
@@ -99,4 +99,18 @@ public List<String> getGroupPerms() {
     public void setGroupPerms(List<String> groupPerms) {
         this.groupPerms = groupPerms;
     }
+
+    @Override
+    public String toString() {
+        return "PlainAccessConfig{" +
```

Comment:
It is awkward and error-prone to manually concatenate strings

Reply:
This toString is generated automatically by the editor I'm using, but for security reasons I've omitted the printing of SecretKey